Code Review 2

caseyjr16 opened the pull request.

hopejoy58 committed:
code review changes

Added new profile icon, created new profile custom class, added get and set methods for username, password, and get for book history for new profile class

hopejoy58 committed:
Merge branch 'wilkinhj-code-review' into spring1-code-review-wilkinhj

vumd-UC committed:
Merge pull request #5 from moburke/spring1-code-review-wilkinhj

Spring1 code review wilkinhj

vumd-UC committed:
Merge pull request #6 from moburke/sprint-1-code-reviews

Sprint 1 code reviews

caseyjr16 committed:
Code Review 1

petromw committed:
Updated main activity including creation of button to start the quiz

petromw committed:
Replaced quiz activity with quiz fragment

petromw committed:
Updated quiz fragment UI

petromw committed:
Updated quiz fragment UI

vumd-UC committed:
Creation of app\data structure for books and questions

moburke committed:
Added new quiz activity, connected it to button 

Added a second quiz activity to connect to button to make sure I could get it to connect

moburke committed:
Delete MainActivity.kt

moburke committed:
Delete MainViewModel.kt

moburke committed:
Delete QuizFragment.kt

moburke committed:
Delete activity_main.xml

moburke committed:
Delete QuizVersion2Activity.kt

moburke committed:
Delete activity_quiz_version2.xml

moburke committed:
added btnStartQuiz activity with new activity

Updated main activity with the button manually

moburke committed:
added quiz activity 2 kt file

Added new quiz activity page to connect to

moburke committed:
added activity quiz xml file

added quiz activity 2 xml file to the main res folder

moburke committed:
commented out old quiz fragment for time being 

I'm just doing this for now until later today.  I'm going to move over the quiz fragment stuff over to the other quiz activity so the same functionality will be there.

moburke committed:
forgot to re-add the xml for the activity quiz

vumd-UC committed:
Added in files for books and questions

caseyjr16 committed:
Code Review 2

caseyjr16 committed:
Merge remote-tracking branch 'origin/master'

vumd-UC committed:
Revert "Merge remote-tracking branch 'origin/master'"

This reverts commit 114cd241014afd6133968f21fbcd9c6867e9535f, reversing
changes made to 0e621e1e68484a26282370cd08dda82cbf01f943.

vumd-UC committed:
Revert "Code Review 2"

This reverts commit 0e621e1e68484a26282370cd08dda82cbf01f943.

caseyjr16 committed:
Code Review 2

Analysis of the program:
• Was the program available in UC Github on time? Yes
• Is the program documented/commented well enough for you to understand? Yes
• Does the program compile? Yes
• Rationale behind your changes? Functionality.

Changes I made:
• Added an linkage functionality to ProfileViewModel.kt. There was no way to link apps prior to this addition. Great way to add on to any ideas to handle incoming links.

Things I learned:
•Click Select Activity from the App Links Assistant.
 Select an activity from the list and click Insert Code.
 The App Links Assistant adds code to your activity's Java file, similar to the following ( Please note: Currently the App Links Assistant does not support Kotlin so you will need to add this code manually):

caseyjr16 committed:
Merge remote-tracking branch 'origin/master'

caseyjr16 commented on app/src/main/java/com/example/whatshouldiread/ui/profile/ProfileViewModel.kt:
```diff
@@ -10,4 +10,31 @@ class ProfileViewModel : ViewModel() {
+                }
+        }
+    }
 }
```
Code Review 2

Analysis of the program:
• Was the program available in UC Github on time? Yes
• Is the program documented/commented well enough for you to understand? Yes
• Does the program compile? Yes
• Rationale behind your changes? Functionality.

Changes I made:
• Added an linkage functionality to ProfileViewModel.kt. There was no way to link apps prior to this addition. Great way to add on to any ideas to handle incoming links.

Things I learned:
•Click Select Activity from the App Links Assistant.
 Select an activity from the list and click Insert Code.
 The App Links Assistant adds code to your activity's Java file, similar to the following ( Please note: Currently the App Links Assistant does not support Kotlin so you will need to add this code manually):

vumd-UC committed:
Revert "Merge remote-tracking branch 'origin/master'"

This reverts commit 96eb740a68ac60248c99ef0e0d5d6d4667225e31, reversing
changes made to 912c31bc1719b4bb194fdf99570cd4fb392d02ad.

vumd-UC committed:
Revert "Code Review 2"

This reverts commit 912c31bc1719b4bb194fdf99570cd4fb392d02ad.

discospiff commented on .idea/codeStyles/css-layout.css:
```diff
@@ -0,0 +1,15 @@
+    </style>
+</resources>
+
+/*
```
Discussion does not be long in source code comments, especially CSS.  This clutters up the code and leads to technical debt.  The best place for discussion is in the pull request.

discospiff commented:
The commits made in this review do not compile.  Only code that compiles should be committed to version control, much less master branch.  

This review was committed to master branch, when it should have been committed to a separate code review branch.  Since this review does not compile, and were committed to the wrong branch, the reviewer should revert the changes from master.

vumd-UC committed:
Merge branch 'Sprint2'

vumd-UC committed:
Revert changes to MainViewModel to fix build

vumd-UC committed:
Remove Profile, add Quiz class and test

vumd-UC committed:
Update book to build when given answers

moburke committed:
Added strings for bookAuthors1-32 and bookName1-32.
Assigned string values to those names and authors.

moburke committed:
Added 3-10 question strings for the ten different answer
options. Added the five questions positions.
Added the getters and setters for answer 3-10.

vumd-UC committed:
Revert "Added 3-10 question strings for the ten different answer"

This reverts commit 12f92926a7d76b49b04642b815d4f97da44b8b24.

vumd-UC committed:
Revert "Added strings for bookAuthors1-32 and bookName1-32."

This reverts commit 90d41747a6855b3af46d0153748f590d49afe432.

vumd-UC committed:
Update quizfragment.xml, revert Kotlin file to original implementation

vumd-UC committed:
Remove QuizVersion2Activity, revert MainActivity back to original implementation

vumd-UC committed:
Update QuizFragment, add in functionality

moburke committed:
Added book names and authors to book folder

vumd-UC committed:
Add in QuizActivity for QuizFragment to run through

moburke committed:
Changed the question data for the quiz

petromw committed:
Created page to display quiz results

moburke committed:
Changed the question data for the quiz part two

vumd-UC committed:
Add in assets folder for questions and book files

vumd-UC committed:
Remove redundant data folder

vumd-UC committed:
Add in functionality for quiz fragment

vumd-UC committed:
Optimize imports

vumd-UC committed:
Merge pull request #12 from moburke/Sprint3

Sprint3 Merge
